### PR TITLE
Display devbox cloud timeout information

### PR DESCRIPTION
**Pull request by LucilleH**
## Summary
Screenshot 2023-03-21 at 11 18 56 AM

## How was it tested?
```
devbox run build
./dist/devbox cloud shell
```

### CODE REVIEW

File: internal/cloud/cloud.go [hunk 1]
```diff
@@ -86,7 +86,8 @@ func ensureVMForUser(vmHostname string, w io.Writer, username string, sshCmd *op
 
 func Shell(ctx context.Context, w io.Writer, projectDir string, githubUsername string) error {
 	color.New(color.FgMagenta, color.Bold).Fprint(w, "Devbox Cloud\n")
-	fmt.Fprint(w, "Remote development environments powered by Nix\n\n")
+	fmt.Fprint(w, "Remote development environments powered by Nix\n")
+	fmt.Fprint(w, "This virtual environment will exit after 8 hours of inactivity\n\n")
```
Comment (gcurtis) [inline internal/cloud/cloud.go:90]: I think we should be explicit that all data will be lost unless it's committed and pushed somewhere else. Exiting sounds like the environment might just shut down but won't be lost.

Comment (Lagoja) [reply internal/cloud/cloud.go:90]: ++

Comment (LucilleH) [reply internal/cloud/cloud.go:90]: All the data is synced to local file system. Do we still have the lost data concern for the CLI scenario? (I hear you on the browser side) @gcurtis @Lagoja

Comment (savil) [inline internal/cloud/cloud.go:90]: could we make it `ux.Fwarning` ?

Comment (LucilleH) [reply internal/cloud/cloud.go:90]: If the user disconnect before syncing completes, their local has all the changes, just the vm doesn't have the latest changes. I don't think users will lose their work tho

**Comment by LucilleH** [comment]
Upon further reflection, user will not lose any data if the vm exits, since the CLI works differently from the web. I'm going to close this pull request.

File: internal/cloud/cloud.go (hunk 1 above)
Comment (gcurtis) [reply internal/cloud/cloud.go:90]: But the user could make some changes on the VM and disconnect before they finish syncing back locally.

**Pull request closed without merging**